server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesRequest.java

```diff
@@ -35,7 +35,7 @@
 
 public final class FieldCapabilitiesRequest extends ActionRequest implements IndicesRequest.Replaceable, ToXContentObject {
     public static final String NAME = "field_caps_request";
-    public static final IndicesOptions DEFAULT_INDICES_OPTIONS = IndicesOptions.strictExpandOpen();
+    public static final IndicesOptions DEFAULT_INDICES_OPTIONS = IndicesOptions.strictExpandOpenAndForbidClosed();
```

Comment:
Nice! I believe this is ok. The doubt I have is just about the fact that we are changing index resolution for the field caps API, and that will affect existing usage. Yet if this is only to avoid errors when targeting closed indices, I think that's the right call, assuming there are no side effects. Are you able to get a green build?

Reply:
Thank you, @javanna, I'll proceed with that

Reply:
I am not 100% sure I am following this. Why are we changing the default behaviour? Don't we want the change only when we get `ignore_unavailable=true`? 
My reasoning here is: if we target a closed index an we have `ignore_unavailable=false` we want an error back, while if the request has `ignore_unavailable=true` we can safely skip closed indices.

Reply:
Oh I see, thank you for the clarification!